app/controllers/questions.js

```diff
@@ -1,55 +1,81 @@
 /**
  * Module dependencies.
  */
-var mongoose = require('mongoose'),
-    async = require('async'),
-    Question = mongoose.model('Question'),
-    _ = require('underscore');
+const mongoose = require('mongoose'),
+  Question = mongoose.model('Question');
+let region;
 
 
 /**
  * Find question by id
+ * @param {object} req
+ * @param {object} res
+ * @param {function} next
+ * @param {string} id
+ * @return {void}
+ *
  */
-exports.question = function(req, res, next, id) {
-    Question.load(id, function(err, question) {
-        if (err) return next(err);
-        if (!question) return next(new Error('Failed to load question ' + id));
-        req.question = question;
-        next();
-    });
+exports.question = (req, res, next, id) => {
+  Question.load(id, (err, question) => {
```

Comment:
convert the mongoose queries and static methods to use promises .exec().then(.....)